### ISSUE TITLE
POST method is not receiving data on android and keeps on LOADING state

### ISSUE THREAD

**Issue by omarDtp** [binaryminds/react-native-sse#35]
I installed react-native-sse and i am using it with method POST
it works well on iOS but on android it doesnt load any data when the method is POST while it works fine in GET
i disabled the Flipper interceptor in debug and its able to establish a connection ( connection opened)
but not recieving any message or other events

react: 18.2.0 => 18.2.0 
    react-native: 0.71.7 => 0.71.7 
    "react-native-sse": "1.2.0",

**Comment by EmilJunker** [comment]
It would be helpful if you could share logs from both iOS and for Android for POST and GET methods so we can compare what is happening. To get logs from react-native-sse, set `debug: true` in EventSource options parameter.

**Comment by omarDtp** [comment]
Resolved the issue with SSE not working with the POST method. The problem stemmed from a performance bottleneck caused by parsing large data with JSON.parse on the Hermes engine, resulting in slower performance compared to the JSC engine.

To address this, we optimized the size of the object being sent via the server, significantly improving performance. Now, we can successfully retrieve data using the POST method.

Thanks for your swift response and support!
